pkg/envoyfilters/envoyfilters.go

```diff
@@ -225,12 +212,42 @@ func CreateVPNConfigPatchFromRule(
 			"value": map[string]interface{}{
 				"name": rbacName,
 				"typed_config": map[string]interface{}{
-					"@type":       "type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC",
-					"stat_prefix": "envoyrbac",
+					"@type": "type.googleapis.com/envoy.extensions.filters.http.rbac.v3.RBAC",
 					"rules": map[string]interface{}{
-						"action":   "ALLOW",
-						"policies": policies,
+						"action": "ALLOW",
+						"policies": map[string]interface{}{
+							shootID + "-inverse": map[string]interface{}{
+								"permissions": []map[string]interface{}{{
+									"not_rule": map[string]interface{}{
+										"header": map[string]interface{}{
```

Comment:
header matcher is duplicated with lines 239 and following, if I see it correctly